Carrot-Assistant/components/embeddings.py

```diff
@@ -14,6 +15,9 @@
 from omop.omop_models import Concept
 
 class EmbeddingModel(Enum):
+    """
+    An Enum for models used to generate concept embeddings
```

Comment:
it would be useful to say why this enum structure is needed. To me it tends to be a little more useful, especially for new people looking at the code.